src/fundus/publishers/de/frankfurter_rundschau.py

```diff
@@ -13,9 +14,15 @@
 
 class FrankfurterRundschauParser(ParserProxy):
     class V1(BaseParser):
-        _paragraph_selector = CSSSelector("p.id-StoryElement-paragraph, article > ul > li")
-        _summary_selector = CSSSelector("p.id-StoryElement-leadText")
-        _subheadline_selector = CSSSelector("h2.id-StoryElement-crosshead")
+        _paragraph_selector = XPath(
+            "//p[@class='id-StoryElement-paragraph'] | "
+            "//p[contains(@class,'id-Article-content-item-paragraph') and text()] |"
+            "//div[@class='id-Article-body']//ul/li[not(@class='id-AuthorList-item ')]"
+        )
+        _summary_selector = CSSSelector(
+            "p.id-StoryElement-leadText, " "p[class='id-Article-content-item id-Article-content-item-summary']"
```

Comment:
There are unnecessary quotation marks

Reply:
Thanks, seems to be left-over from formatting.